scripts/py_matter_idl/matter_idl/test_xml_parser.py

```diff
@@ -223,6 +220,62 @@ def testFabricScopedAndSensitive(self):
                                       qualities=StructQuality.FABRIC_SCOPED)],
                          )]))
 
+    def testEnum(self):
+        idl = XmlToIdl('''<?xml version="1.0"?>
+            <configurator>
+              <cluster><name>Test1</name><code>0x000A</code></cluster>
+              <cluster><name>Test2</name><code>20</code></cluster>
```

Comment:
We should not show two different number formats...

Reply:
Why wouldn't the tools support both formats and test support for both formats?

Reply:
This is copy and paste from a different test. I do not think this matters for this particular test.